solidity/hardhat.config.ts

```diff
@@ -30,7 +31,7 @@ const config: HardhatUserConfig = {
         url: process.env.FORKING_URL || "",
         // latest block is taken if FORKING_BLOCK env is not provided
         blockNumber:
-          process.env.FORKING_BLOCK && parseInt(process.env.FORKING_BLOCK),
+          process.env.FORKING_BLOCK && parseInt(process.env.FORKING_BLOCK, 10),
```

Comment:
Why do we need that `10`? I am looking at example [in coverage pools](https://github.com/keep-network/coverage-pools/blob/e85b7d624368eaf38c0405ae6b225e3be3bcc4d9/hardhat.config.ts#L31-L33) and in [threshold network contracts](https://github.com/threshold-network/solidity-contracts/pull/22/files#diff-57cf5378f8cab20b02b279097ba6fad08eb53f747d81770045dda9c5f95effe3R29-R31) and I do not see anything similar.

Reply:
On the other we used it in https://github.com/keep-network/keep-core/blob/874e247e8f21cec976f93b38604c37a1f77b8872/solidity/random-beacon/hardhat.config.ts#L38.